spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -168,11 +167,12 @@ public synchronized void onJobStart(SparkListenerJobStart jobStart) {
         super.onJobStart(jobStart);
         int jobId = jobStart.jobId();
         String jobGroupId = jobStart.properties().getProperty("spark.jobGroup.id");
+        String uiEnabled = jobStart.properties().getProperty("spark.ui.enabled");
         String jobUrl = getJobUrl(jobId);
         String noteId = Utils.getNoteId(jobGroupId);
         String paragraphId = Utils.getParagraphId(jobGroupId);
           
-        if (jobUrl != null && noteId != null && paragraphId != null) {
+        if (uiEnabled != "false" && jobUrl != null) {
```

Comment:
does this need to check for string content, eg. trim whitespaces, lower case etc? since this is user input?